library/templates/dcp_censustracts.yml

```diff
@@ -0,0 +1,33 @@
+dataset:
+  name: &name dcp_censustracts
+  version: "{{ version }}"
+  acl: public-read
+
+  source:
+    url: 
+      path: https://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nyct2010_{{ version }}.zip
+      subpath: nyct2010_{{ version }}/nyct2010.shp
+    geometry:
+      SRS: EPSG:2263
+      type: MULTIPOLYGON
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+  
+  destination:
+    name: *name
+    geometry:
+      SRS: EPSG:4326
+      type: MULTIPOLYGON
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+    fields: []
+    sql: null
+
+  info:
+    description: |
+      ### ESRI Shapefile
+      The Census Tracts for the 2010 US Census (Clipped to shoreline).
```

Comment:
We will have to remember to switch to 2020 in this description eventually. That or remove the date.

Reply:
@SPTKL can I get your opinion on this, too?